tests/example.py

```diff
@@ -6,14 +6,14 @@
 
 if platform.system() == 'Windows':
     libPath = libPath.replace('\\tests',
-                              '\\src\\latis-python-client')
+                              '\\src\\latispy')
 else:
     libPath = libPath.replace('/tests',
-                              '/src/latis-python-client')
+                              '/src/latispy')
 
 sys.path.insert(0, libPath)
 
-import latis
+import latispy as latis
```

Comment:
latispy may be a bit redundant? Perhaps just latis would be a better valid module name?